Scripts/DCS-BIOS/lib/common/Functions.lua

```diff
@@ -25,6 +25,15 @@ function Functions.pad_left(str, len)
 	return string.rep(" ", len - #str) .. str
 end
 
+--- @func Constructs a string of the specified length with the right padded by whitespace if necessary.
+--- @param str string? The base text
+--- @param len number The length the string should be
+--- @return string result A new string of length len, with whitespace padding added to the right as necessary
+function Functions.pad_right(str, len)
+	str = tostring(Functions.coerce_nil_to_string(str))
```

Comment:
The `tostring` call here is unnecessary as `Functions.coerce_nil_to_string` returns, well... a string
```suggestion
	str = Functions.coerce_nil_to_string(str)
```

Reply:
fixed